compose/config/types.py

```diff
@@ -136,6 +136,20 @@ def normalize_path_for_engine(path):
     return path.replace('\\', '/')
 
 
+def normpath(path, win_host=False):
+    """ Custom path normalizer that handles Compose-specific edge cases like
+        UNIX paths on Windows hosts and vice-versa. """
+
+    sysnorm = ntpath.normpath if win_host else os.path.normpath
```

Comment:
Shouldn't this be:
```
-sysnorm = ntpath.normpath if win_host else os.path.normpath
+sysnorm = ntpath.normpath if win_host else posix.normpath
```

Reply:
`win_host` will be true only if `COMPOSE_FORCE_WINDOWS_HOST` is set. In this case we want to produce NT style paths regardless of the client platform. In all other cases, we want to use the default normalization for the client system, hence the use of `os.path.normpath`